### DISCUSSION TITLE
How do I hide players so that they cannot be selected in Music Assistant

### DISCUSSION THREAD

**Discussion by rcjkierkels** [music-assistant/support#1605] [Q&A] [answered]
I want Music Assistant only to be enabled for certain media player groups. Now it is showing all mediaplayers in my network. Is there a configuration option that I can choose which mediaplayers can be used with music assistant?

**Comment by OzGav** [comment, marked as answer]
Go to MA SETTIMGS >>PLAYERS and select the player you don’t want to see and select DISABLE.
